projects/elife.yaml

```diff
@@ -495,6 +496,12 @@ journal:
                     fallbacks:
                         4xx: "4xx.html"
                         5xx: "5xx.html"
+                vcl-templates:
+                    journal-submit:
+                        percentage: 100
+                        referer: ^(?:https://end2end--xpub\.elifesciences\.org/|https://end2end--profiles\.elifesciences.org:8082/)
+                        referer: ^https://end2end--xpub\.elifesciences\.org/
```

Comment:
@giorgiosironi Duplicate entry. Don't need the first though as Profiles/ORCID doesn't have anything for the user to interact with?

Reply:
Latter's applied and is apparently working.